src/components/molecules/NavBar/NavBar.scss

```diff
@@ -270,4 +272,30 @@ $border-radius: 28px;
   &__profile-popover {
     height: 100%;
   }
+
+  &__photobooth {
+    &-button {
+      position: absolute;
+      top: 130px;
+      left: -($spacing--lg);
+      border-radius: 0px $border-radius--md $border-radius--md 0px;
+      padding: $spacing--xs $spacing--md;
+      cursor: pointer;
+      background-color: $lighter-intermediate-grey;
+      background-image: url(/assets/icons/photobooth-icon.png);
+      background-size: $photobooth-button-background-size;
+      background-position: 50% 10%;
+      background-repeat: no-repeat;
+
+      &:hover {
+        background-color: $light-grey;
+      }
+    }
+
+    &-title {
```

Comment:
```suggestion
    &--title {
```

Reply:
Title isn't modifier here. The & is making it a bit more complex to notice that it's actually part of a more complex name.

`NavBar__photobooth-title`

My suggestion would be to just unroll this case as `&__photobooth-title` without the double `&` concatenation.

Reply:
done b230401